website/docs/fun/counting_v2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Counting System
+---
+
+This command controls the counting game and the statistics command.
+
+This version of a counting command allows for the usage of base 10 and roman numerals, optionally joined by the usage of math functions and/or operators to count from 1, incrementing by 1, until failure.
+
+Also included are statistics for the server and each user for how many times they have counted and how many times they've been correct.
+
+Messages without text are ignored by the system, effectively allowing unrelated content. However, messages including text which cannot be translated into a math expression will be quoted as invalid, but not deleted.
```

Comment:
I think this should be in a `::note` callout for visibility. Concerning the content, I suggest
```suggestion
Unlike some other counting systems, this command does not delete invalid numbers to allow counting and other conversation to take place concurrently. Messages that include text which do not correspond to valid numbers will, however, trigger a warning message.
```

A clarifying question: is it the case that completely unrelated messages that do not contain anything resembling a number or a mathematical expression (so something like `Hello there!`) trigger the invalid input warning as well? If so, I would think this would disrupt conversation to the extent that you might as well disallow invalid inputs completely and delete them as most counting games do. But if you feel strongly that this command is fine as is, I am happy to drop this point.

Reply:
Yes, messages such as `Hello there!` will trigger the warning message. I agree with you though, and will work on allowing messages as such without warning for them.

Reply:
Messed around with the expression regex and trying to find logic that would determine the message is not meant to be an expression. Issue is the regex, due to the part which captures roman numerals, also captures empty strings, which I'd prefer it not but cannot figure out how to require at least one roman numeral (I, V, X, L, C, D, M) for it to be captured as a roman numeral. Another issue is, using the logic `if eq (joinStr "" $re) ""`, if the message includes the common letter e, or another character related to math, it evaluates to false, therefore disqualifying that method.

Reply:
Update: As of the latest commit - which makes a change to the regex which parses the expression and adds a conditional statement to make sure an expression was parsed - I believe this command to act fully as intended. 

Messages which are not meant to be expressions are to be ignored, message which are attempted expressions but have invalid syntax or include unaccepted characters/"functions" are to be given a warning message, and valid expressions are to be accepted and evaluated towards the correct next number.